config_example.ini

```diff
@@ -4,19 +4,6 @@ secret = "HOOK_SECRET"
 github_username = "GITHUB_USERNAME"
 github_access_token = "SECRET_TOKEN"
 
-; Twitter auth details
```

Comment:
We can keep these config examples for both Slack and Twitter because the PHP code is not deleted yet and just not enabled. We may need them one day.

Reply:
I've restored the file